record-and-playback/core/lib/recordandplayback/edl/audio.rb

```
@@ -223,16 +332,17 @@ def self.process_segment(entry, output_basename, i, total_segments, audioinfo)
 
               # Build track label
               track_label = "t#{i}_#{idx}"
-              line = "[#{input_index}]#{FFMPEG_AFORMAT},apad"
-              line << ",atempo=#{speed},atrim=start=#{ms_to_s(audio_data[:timestamp])}" if speed != 1.0
-              line << ",asetpts=N[#{track_label}];"
+              line = "[#{input_index}]"
+              line << "atempo=#{speed},atrim=start=#{ms_to_s(audio_date[:timestamp])}," if speed != 1.0
```

RuboCop (1.85.1)

[warning] Avoid inequality comparisons of floats as they are unreliable.

(Lint/FloatComparison)